### PR TITLE
fix: social icons layout bug

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What changes did you make? (Give an overview)

Fixed the layout bug - https://github.com/eslint/eslint.org/pull/381#issuecomment-1349514328.

| before | After |
|--------|------|
|  image | image

### CODE REVIEW

**Review by harish-sethuraman — COMMENTED** [review]
Let's fix this first at main ESLint docs and then lets make the same fix here.

**Comment by Tanujkanti4441** [comment]
Screenshot 2022-12-15 002807

the language select is also going out of the screen in homepage.

File: src/assets/scss/components/social-icons.scss [hunk 1]
```diff
@@ -10,7 +10,6 @@
         padding: 0;
         margin-left: -1rem;
         margin-inline-start: -1rem;
```
Comment (shanpriyan) [inline src/assets/scss/components/social-icons.scss:12]: if we do the following changes we don't need to wrap the mastodon icon , it appears in the same line even at different breakpoints.
<details>
<summary>Screenshot (1052 x 522)</summary>


image


</details>

```suggestion
        display: flex;
        justify-content: space-between;
```


Remove `padding` from here 

https://github.com/eslint/eslint.org/blob/885d0d939ebc77370f56db475c750893552335e4/src/assets/scss/components/social-icons.scss#L22

Comment (nzakas) [reply src/assets/scss/components/social-icons.scss:12]: How does that work on the homepage footer?

**Comment by harish-sethuraman** [comment]
@Tanujkanti4441  I guess it is fixed in this PR's preview right?

**Comment by Tanujkanti4441** [comment]
> 

ya that select one is fixed in this PR my bad.

problem is all about the wrap part

Screenshot 2022-12-15 223031

**Comment by Tanujkanti4441** [comment]
Adding these lines of code to the [src/assets/scss/components/social-icons.scss](https://github.com/eslint/eslint.org/pull/382/files/e372b7b6ba393b27e85b0ee24dab5c875193265e#diff-4dd655cea29886ba0c299b787f030ac980a50b53d8c176e5b458894404a1eae5) file is solving the problem for me.
Screenshot 2022-12-15 233615


please tell me if is it ok to sharing the code by a screenshot rather than the code pen

File: src/assets/scss/components/social-icons.scss (hunk 1 above)
Comment (shanpriyan) [reply src/assets/scss/components/social-icons.scss:12]: we need to change the properties little bit to achieve that.

Comment (shanpriyan) [reply src/assets/scss/components/social-icons.scss:12]: Added nowrap changes with socials layout fix in https://github.com/eslint/eslint.org/pull/386 (homepage and playground)

**Comment by amareshsm** [comment]
> Adding these lines of code to the [src/assets/scss/components/social-icons.scss](https://github.com/eslint/eslint.org/pull/382/files/e372b7b6ba393b27e85b0ee24dab5c875193265e#diff-4dd655cea29886ba0c299b787f030ac980a50b53d8c176e5b458894404a1eae5) file is solving the problem for me.
> 
> Screenshot 2022-12-16 013348
> 
> please let me know if it is ok to sharing the code by a screenshot rather than the code pen

@Tanujkanti4441  Thanks for the suggestion. it's similar to https://github.com/eslint/eslint.org/pull/386.

**Comment by harish-sethuraman** [comment]
Closing in favour of https://github.com/eslint/eslint.org/pull/386

**Pull request closed without merging**